tests/unit/cacheEvictionTest.js

```diff
@@ -1,4 +1,4 @@
-import AsyncStorage from '@react-native-async-storage/async-storage';
+import Storage from '../../lib/storage';
```

Comment:
So, this will use the `AsyncStorage` not `localforage`? Does `localforage` have mocks and should we write tests that show the providers are equivalent?

Reply:
>We should write tests that show the providers do what they are intended to do

Cool. I think it would be good to write some. But not a blocker for me if this is otherwise ready to get merged. I guess I just have a vague concern that by having two storage implementations we need to remember to always keep the interface the same and make sure we are not testing changes against only one provider, but I don't know if that will be an issue in practice.